LanguageFeatures/Patterns/Exhaustiveness/lifting_cast_pattern_A01_t01.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion The lifted space union for a pattern with matched value type M is
+/// ...
+/// Cast pattern:
+/// The space union spaces for a cast pattern with cast type C is a union of:
+/// - The lifted space union of the cast's subpattern in context C.
+/// - For each space E in the expanded spaces of M:
+///   a. If E is not a subset of C and C is not a subset of M, then the lifted
+///     space union of E.
+///
+/// @description Check a lifted space of a cast pattern in case of not sealed
+/// type
```

Comment:
The point would be that these switches _are_ actually exhaustive, but the exhaustiveness algorithm can't see it because it has no space modeling "everything which is not `int`".

Could the description say something like that, in addition to what it says currently?

Reply:
Waiting for https://github.com/dart-lang/co19/issues/2010#issuecomment-1498779364